api/python/quilt3/data_transfer.py

```diff
@@ -85,6 +85,72 @@ class S3ClientProvider:
 
     We assume that public buckets are read-only: write operations should always use S3ClientProvider.standard_client
     """
+    _event_callbacks = {}
```

Comment:
in [docs](https://boto3.amazonaws.com/v1/documentation/api/latest/guide/events.html) they are called "event handlers" I guess it worth to follow this terminology to avoid confusion

Reply:
changed